lib/Translation/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php

```diff
@@ -223,28 +223,28 @@ protected function addBlocks(
         foreach ($fieldDefinitions as $fd) {
             if ($fd instanceof DataObject\ClassDefinition\Data\Block) {
 
-                /** @var $blockLocalizedFieldDefinition DataObject\ClassDefinition\Data\Localizedfields */
+                /** @var DataObject\ClassDefinition\Data\Localizedfields $blockLocalizedFieldDefinition */
                 $blockLocalizedFieldDefinition = $fd->getFielddefinition('localizedfields');
                 if ($blockLocalizedFieldDefinition) {
                     $blockLocalizedFieldsDefinitions = $blockLocalizedFieldDefinition->getFieldDefinitions();
 
-                    /** @var $blockItems array */
+                    /** @var array $blockItems */
                     $blocks = $object->{'get' . ucfirst($fd->getName())}();
 
                     if ($blocks) {
-                        /** @var $blockItem DataObject\Data\BlockElement */
+                        /** @var DataObject\Data\BlockElement $blockItem */
```

Comment:
$blockItems

Reply:
I think this was correct, but on the wrong line. I changed it